core/src/utils/overlays-interface.ts

```diff
@@ -8,7 +8,7 @@ export interface OverlayEventDetail<T = any> {
 }
 
 export interface OverlayInterface {
-  el: HTMLElement;
+  el: HTMLIonOverlayElement;
```

Comment:
This should always have been `HTMLIonOverlayElement`. Without it, I get typescript errors when trying to access `lastFocus` (since it is only known property of HTMLIonOverlayElement not HTMLElement)